OnBaseDocsApi/api-config-example.yaml

```diff
@@ -4,3 +4,10 @@ dataSource: testDatabaseName
 serviceUrl: https://onbase-url.com
 docIndexKeyName: Index Key Sample
 stagingDocType: Staging Sample
+profiles:
+  profile1:
+    username: user
+    password: pass
+  profile2:
+    username: user2
+    password: pass2
```

Comment:
Besides these, we also need fields for basic auth and ports (normal and admin port): https://github.sig.oregonstate.edu/ecs-data/configs/blob/32a1785e05f752b77ee43cc3aff5f70cb4b3e0cb/onbase-api-v1-dev/default.yaml#L4-L13

Reply:
Done.